### PR TITLE
Update the browser agent update page

### PR DESCRIPTION
**Pull request by metal-messiah**
## Give us some context

This instructional document is out of date and not relevant for newer Browser Agent builds.  This doc attempts to give clear instructions for both older versions that are still in our EOL timeline, as well as the newer builds that were previously not described in this doc.

### CODE REVIEW

**Comment by metal-messiah** [comment]
Sidekicks: @patrickhousley @bjfield

File: src/content/docs/browser/new-relic-browser/installation/update-browser-agent.mdx [hunk 1]
```diff
@@ -16,21 +16,41 @@ Running the latest version of our browser agent ensures you can access all brows
...
-
-There are different [agent types](/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent#agent-types). The Pro+SPA agent has the format `js-agent.newrelic.com/nr-spa-1184.min.js`
-
-To verify the latest version of the browser script loader:
-
-1. Go to [js-agent.newrelic.com/nr-loader-full-current.min.js](https://js-agent.newrelic.com/nr-loader-full-current.min.js) .
-2. Search for `js-agent.newrelic.com/nr-`, then note the numbers that follow `nr-`.
-
-If the latest version number is higher than the number of the version you are currently running, update your browser agent.
+Verifying your browser version number can be done by one of the following:
```
Comment (a-sassman) [inline src/content/docs/browser/new-relic-browser/installation/update-browser-agent.mdx:19]:
```suggestion
To verify your browser version number, try one of the following:
```

File: src/content/docs/browser/new-relic-browser/installation/update-browser-agent.mdx [hunk 2]
```diff
@@ -16,21 +16,41 @@ Running the latest version of our browser agent ensures you can access all brows
...
+1. Inspect the agent harvest payloads (`Any Browser Agent Version`).
```
Comment (a-sassman) [inline src/content/docs/browser/new-relic-browser/installation/update-browser-agent.mdx:20]:
```suggestion
* Inspect the agent harvest payloads (`Any Browser Agent Version`).
```

File: src/content/docs/browser/new-relic-browser/installation/update-browser-agent.mdx [hunk 3]
```diff
@@ -16,21 +16,41 @@ Running the latest version of our browser agent ensures you can access all brows
...
+   1. Open the developer tools of your browser.
+   2. Open the view in the developer tools that shows network traffic.
+   3. Filter the results to only those that include `nr-data.net`.
+   4. The version will be included in the query parameters of each call to the `nr-data.net` endpoint, in the format of `?v=<version>`.
+2. Inspect the source code of the agent snippet.
```
Comment (a-sassman) [inline src/content/docs/browser/new-relic-browser/installation/update-browser-agent.mdx:25]:
```suggestion
* Inspect the source code of the agent snippet:
```

File: src/content/docs/browser/new-relic-browser/installation/update-browser-agent.mdx [hunk 4]
```diff
@@ -16,21 +16,41 @@ Running the latest version of our browser agent ensures you can access all brows
...
+    * `v1217` and Newer:
```
Comment (a-sassman) [inline src/content/docs/browser/new-relic-browser/installation/update-browser-agent.mdx:26]:
```suggestion
    * For`v1217` and later:
```

File: src/content/docs/browser/new-relic-browser/installation/update-browser-agent.mdx [hunk 5]
```diff
@@ -16,21 +16,41 @@ Running the latest version of our browser agent ensures you can access all brows
...
+      1. View the source code of a page which includes the JavaScript snippet.
+      2. Search the page source for `PROD` -- The version will be prepended, in the format of `<version>.PROD`.
+    * `v1216` and Older:
```
Comment (a-sassman) [inline src/content/docs/browser/new-relic-browser/installation/update-browser-agent.mdx:29]:
```suggestion
    * For `v1216` or older:
```

File: src/content/docs/browser/new-relic-browser/installation/update-browser-agent.mdx [hunk 6]
```diff
@@ -16,21 +16,41 @@ Running the latest version of our browser agent ensures you can access all brows
...
+      3. View the source code of a page which includes the JavaScript snippet.
```
Comment (a-sassman) [inline src/content/docs/browser/new-relic-browser/installation/update-browser-agent.mdx:30]:
```suggestion
      1. View the source code of a page which includes the JavaScript snippet.
```

File: src/content/docs/browser/new-relic-browser/installation/update-browser-agent.mdx [hunk 7]
```diff
@@ -16,21 +16,41 @@ Running the latest version of our browser agent ensures you can access all brows
...
+      4. Search the page source for `js-agent.newrelic.com/nr-`.
```
Comment (a-sassman) [inline src/content/docs/browser/new-relic-browser/installation/update-browser-agent.mdx:31]:
```suggestion
      2. Search the page source for `js-agent.newrelic.com/nr-`.
```

File: src/content/docs/browser/new-relic-browser/installation/update-browser-agent.mdx [hunk 8]
```diff
@@ -16,21 +16,41 @@ Running the latest version of our browser agent ensures you can access all brows
...
+      5. The numbers that follow `nr-` are your current version. For example, `js-agent.newrelic.com/nr-593.min.js` indicates you are running version `593` of the browser script.
```
Comment (a-sassman) [inline src/content/docs/browser/new-relic-browser/installation/update-browser-agent.mdx:32]:
```suggestion
      3. The numbers that follow `nr-` are your current version. For example, `js-agent.newrelic.com/nr-593.min.js` indicates you are running version `593` of the browser script.
```

File: src/content/docs/browser/new-relic-browser/installation/update-browser-agent.mdx [hunk 9]
```diff
@@ -16,21 +16,41 @@ Running the latest version of our browser agent ensures you can access all brows
...
+3. Inspect the newrelic global object (v1224+).
```
Comment (a-sassman) [inline src/content/docs/browser/new-relic-browser/installation/update-browser-agent.mdx:33]:
```suggestion
* Inspect the newrelic global object (v1224 or higher):
```

File: src/content/docs/browser/new-relic-browser/installation/update-browser-agent.mdx [hunk 10]
```diff
@@ -16,21 +16,41 @@ Running the latest version of our browser agent ensures you can access all brows
...
+There are different [agent types](/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent#agent-types). The Pro+SPA agent loader files use the format `js-agent.newrelic.com/nr-loader-<type>-<version>.min.js`
+
+To verify the version of the browser script loader on your page:
+1. Inspect the newrelic global object (v1224+).
+   1. Open the developer tools of your browser.
+   2. Open the console view.
+   3. In the console, enter `newrelic.initializedAgents`.
+   4. In the resulting object, expand the relevant agent object.
+   5. Inside the relevant agent object, find `runtime.loaderType`.
+
```
Comment (a-sassman) [inline src/content/docs/browser/new-relic-browser/installation/update-browser-agent.mdx:50]:
```suggestion
To verify the version of the browser script loader on your page, inspect the newrelic global object (v1224 or higher):
   1. Open the developer tools of your browser.
   2. Open the console view.
   3. In the console, enter `newrelic.initializedAgents`.
   4. In the resulting object, expand the relevant agent object.
   5. Inside the relevant agent object, find `runtime.loaderType`.

```

File: src/content/docs/browser/new-relic-browser/installation/update-browser-agent.mdx [hunk 11]
```diff
@@ -16,21 +16,40 @@ Running the latest version of our browser agent ensures you can access all brows
...
+   1. Open the developer tools of your browser.
+   2. Open the console view.
+   3. In the console, enter `newrelic.initializedAgents`.
+   4. In the resulting object, expand the relevant agent object.
+   5. Inside the relevant agent object, find `runtime.version`.
+
+## Check your installation type
+There are different [agent types](/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent#agent-types). The Pro+SPA agent loader files use the format `js-agent.newrelic.com/nr-loader-<type>-<version>.min.js`
+
+To verify the version of the browser script loader on your page, inspect the newrelic global object (v1224 or higher):
```
Comment (a-sassman) [inline src/content/docs/browser/new-relic-browser/installation/update-browser-agent.mdx:43]:
```suggestion
There are different [agent types](/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent#agent-types). The Pro+SPA agent loader files use the format `js-agent.newrelic.com/nr-loader-<type>-<version>.min.js`. 
To verify the version of the browser script loader on your page, inspect the newrelic global object (v1224 or higher):
```

File: src/content/docs/browser/new-relic-browser/installation/update-browser-agent.mdx [hunk 12]
```diff
@@ -16,21 +16,39 @@ Running the latest version of our browser agent ensures you can access all brows
...
+## Check your installation type
+There are different [agent types](/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent#agent-types). The Pro+SPA agent loader files use the format `js-agent.newrelic.com/nr-loader-<type>-<version>.min.js`. 
+To verify the version of the browser script loader on your page, inspect the newrelic global object (v1224 or higher):
+   1. Open the developer tools of your browser.
+   2. Open the console view.
+   3. In the console, enter `newrelic.initializedAgents`.
+   4. In the resulting object, expand the relevant agent object.
+   5. Inside the relevant agent object, find `runtime.loaderType`.
+
+## To check if your version is out of date
```
Comment (a-sassman) [inline src/content/docs/browser/new-relic-browser/installation/update-browser-agent.mdx:49]:
```suggestion
## Check if version is outdated
```

File: src/content/docs/browser/new-relic-browser/installation/update-browser-agent.mdx [hunk 13]
```diff
@@ -16,21 +16,39 @@ Running the latest version of our browser agent ensures you can access all brows
...
+## Check your installation type
+There are different [agent types](/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent#agent-types). The Pro+SPA agent loader files use the format `js-agent.newrelic.com/nr-loader-<type>-<version>.min.js`. 
+To verify the version of the browser script loader on your page, inspect the newrelic global object (v1224 or higher):
+   1. Open the developer tools of your browser.
+   2. Open the console view.
+   3. In the console, enter `newrelic.initializedAgents`.
+   4. In the resulting object, expand the relevant agent object.
+   5. Inside the relevant agent object, find `runtime.loaderType`.
+
+## Check if version is outdated
```
Comment (a-sassman) [inline src/content/docs/browser/new-relic-browser/installation/update-browser-agent.mdx:49]:
```suggestion
## Check if version is outdated

To verify if your browser version if outdated:
```

File: src/content/docs/browser/new-relic-browser/installation/update-browser-agent.mdx [hunk 14]
```diff
@@ -16,21 +16,41 @@ Running the latest version of our browser agent ensures you can access all brows
...
```
Comment (a-sassman) [inline src/content/docs/browser/new-relic-browser/installation/update-browser-agent.mdx:49]:
```suggestion
## Check if your version is outdated
```

**Review by a-sassman — APPROVED** [review]

File: src/content/docs/browser/new-relic-browser/installation/update-browser-agent.mdx [hunk 15]
```diff
@@ -16,21 +16,41 @@ Running the latest version of our browser agent ensures you can access all brows
...
+   1. Open the developer tools of your browser.
+   2. Open the console view.
+   3. In the console, enter `newrelic.initializedAgents`.
+   4. In the resulting object, expand the relevant agent object.
+   5. Inside the relevant agent object, find `runtime.loaderType`.
+
+## Check if your version is outdated
+
+To verify if your browser version if outdated:
+   1. See the [browser agent end of life](https://js-agent.newrelic.com/nr-loader-full-current.min.js) table.
```
Comment (metal-messiah) [inline src/content/docs/browser/new-relic-browser/installation/update-browser-agent.mdx:52]: Just noticed, this link is incorrect.

```suggestion
   1. See the [browser agent end of life](https://docs.newrelic.com/docs/browser/browser-monitoring/getting-started/browser-agent-eol-policy/) table.
```

**Comment by metal-messiah** [comment]
> @metal-messiah Is this ready to be published?

@a-sassman  I found one small mistake, and fixed.  Should be good to go now!  Thanks for the help on this!

File: src/content/docs/browser/new-relic-browser/installation/update-browser-agent.mdx [hunk 16]
```diff
@@ -16,21 +16,41 @@ Running the latest version of our browser agent ensures you can access all brows
...
+      2. Search the page source for `js-agent.newrelic.com/nr-`.
+      3. The numbers that follow `nr-` are your current version. For example, `js-agent.newrelic.com/nr-593.min.js` indicates you are running version `593` of the browser script.
+* Inspect the newrelic global object (v1224 or higher):
+   1. Open the developer tools of your browser.
+   2. Open the console view.
+   3. In the console, enter `newrelic.initializedAgents`.
+   4. In the resulting object, expand the relevant agent object.
+   5. Inside the relevant agent object, find `runtime.version`.
+
+## Check your installation type
```
Comment (a-sassman) [inline src/content/docs/browser/new-relic-browser/installation/update-browser-agent.mdx:40]:
```suggestion
## Check your installation type

```

**Review by a-sassman — APPROVED** [review]

**Pull request merged**